tests/distributions/test_continuous.py

```diff
@@ -562,6 +576,18 @@ def test_studentt_logcdf(self):
             lambda value, nu, mu, sigma: st.t.logcdf(value, nu, mu, sigma),
         )
 
+    def test_studentt_icdf(self):
+        check_icdf(
+            pm.StudentT,
+            {"nu": Rplusbig, "mu": R, "lam": Rplus},
+            lambda q, nu, mu, lam: st.t.ppf(q, nu, mu, lam**-0.5),
+        )
+        check_icdf(
```

Comment:
No need for both parametrization tests. That's something we used to do for logp but makes no sense really, unless the expression we used depended on the parametrization

Reply:
Forgot to change it in https://github.com/pymc-devs/pymc/commit/be24e627a698659037db8cb8affb9c82ad24d2ee. Fixed in https://github.com/pymc-devs/pymc/pull/6845/commits/fdfe4a1de8dc59293e8ab1e935780f5f7257f2e3.